.coderabbit.yaml

```
@@ -0,0 +1,176 @@
+    - "yarn.lock"                   # yarn.lock 무시
+    - "pnpm-lock.yaml"              # pnpm-lock.yaml 무시
+    - ".eslintrc.*"                 # ESLint 설정 무시
+    - "tsconfig.json"               # TypeScript 설정 무시
+    - ".prettierrc.*"               # Prettier 설정 무시
+    - "next.config.*"               # Next.js 설정 무시
+
+# AI 프롬프트 설정
+prompts:
+  default: |
+    당신은 Next.js 14.2.26, TypeScript, zustand, supabase, react-hook-form, zod, shadcn ui, tanstack query에 전문성을 가진 리뷰어입니다.
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)